Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -246,6 +246,12 @@ public final class SwiftTargetBuildDescription {
     /// Whether to disable sandboxing (e.g. for macros).
     private let shouldDisableSandbox: Bool
 
+    /// Whether to build preparing for indexing
+    public var prepareForIndexing: Bool {
+        // Do full build for tools
+        defaultBuildParameters.prepareForIndexing && target.buildTriple != .tools
```

Comment:
If the `target` triple ` is `tools` then `defaultBuildParameters` are going to be `toolsBuildParameters` which wouldn't have this flag anyway?

Reply:
That is the purpose of the check for `.tools`. This check will always be false if the `buildTriple` is `.tools`. I can probably make that cleaner. I should also make sure `prepareForIndexing` is false in `toolsBuildParameters` though I never check that anywhere.

Reply:
Ah, I see what you are saying. I don't need the check if I make the `toolsBuildParameters.prepareForIndexing` set to false.

Reply:
Indeed.

Reply:
Made sure the toolsBuildParameters are set properly so I don't need this getter at all any more.